.github/workflows/build_and_test.yml

```diff
@@ -177,12 +179,14 @@ jobs:
         run: tar -xzf /tmp/storybook-static-e2e.tar.gz
 
       - name: Run e2e tests
-        run: yarn --cwd packages/react-component-library test:e2e --shard ${{ matrix.shard }}
+        env:
+          NODE_OPTIONS: --dns-result-order=ipv4first
```

Comment:
Scoped `NODE_OPTIONS` to the specific command and not the whole workflow, as it started to affect actions used in other steps otherwise (as they use Node.js as well – some were on old versions of Node.js which don't support `--dns-result-order`).